ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -1233,6 +1233,31 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: azure-ipi-confidentialvm-vmgueststateonly-f5
```

Comment:
Suggest to add some y-stream and z-stream upgrade job in openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.18.yaml and openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.17.yaml

Reply:
Added in upgrade job openshift-openshift-tests-private-release-4.18__amd64-nightly-4.18-upgrade-from-stable-4.18.yaml.
When bug [OCPBUGS-36670](https://issues.redhat.com/browse/OCPBUGS-36670) backported to 4.17, will add y-stream upgrade job.